### PR TITLE
Complete invoice class and invoice_repo_class

### PR DESCRIPTION
**Pull request by suzkiee**
Hey everyone, this PR finishes out the tests for the invoice class and the invoice_repo. 

Questions
none today 

To Review: 
Can i get a second glance over to make sure: 
All methods were present? 
All methods were tested?

### CODE REVIEW

File: lib/invoice_repo.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
+require 'CSV'
+require 'time'
+require 'invoice'
+
+class InvoiceRepo
```
[the remark this answers is not in the document]
Comment (JoannaCoPo) [reply lib/invoice_repo.rb:5]: Following up on convention for this

File: lib/invoice_repo.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  def find_by_id(id)
+    @invoices.find do |invoice|
+      invoice.id == id
+    end
+  end
+
+  def create(attributes)
+    invoice = Invoice.new(attributes, self)
+    max = @invoices.max_by do |invoice|
```
[the remark this answers is not in the document]
Comment (JoannaCoPo) [reply lib/invoice_repo.rb:35]: investigating this

File: lib/invoice_repo.rb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+class InvoiceRepo
+  attr_reader :invoices
+  def initialize(path, engine)
+    @invoices = []
+    @engine = engine
+    populate_information(path)
+  end
+
+  def populate_information(path)
+    CSV.foreach(path, headers: true, header_converters: :symbol) do |invoice_info|
```
[the remark this answers is not in the document]
Comment (JoannaCoPo) [reply lib/invoice_repo.rb:14]: adjusted locally

File: lib/invoice_repo.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (JoannaCoPo) [reply lib/invoice_repo.rb:5]: investigating this

File: lib/invoice_repo.rb [hunk 4]
```diff
@@ -0,0 +1,72 @@
+require 'CSV'
```
[the remark this answers is not in the document]
Comment (JoannaCoPo) [reply lib/invoice_repo.rb:1]: investigating this

**Comment by suzkiee** [comment]
Closing to recreate PR

**Pull request closed without merging**